server.go

```diff
@@ -63,6 +68,8 @@ const (
 	haproxyConfig        = "haproxy.cfg"
 	haproxyPid           = "haproxy.pid"
 	appPid               = "PID_FILE"
+	haProxyPxnameIndex   = 0
```

Comment:
I know haproxy's label for this field is "pxname" but it's not obvious what that is - could you document it either in a comment or by changing the variable name